lib/descs-to-fields.js

```diff
@@ -169,11 +169,13 @@ const descsToFields = (descs, resolveMiddlewares = () => {}) => {
     finish = resolveMiddlewares(cloneDeep(req))
     req = {}
   })
-  return mapValues(descs, (desc, key) => ({
+  return mapValues(descs, (desc) => ({
     type: descToType(desc),
     args: descToArgs(desc),
     description: desc.description || '',
     resolve: (source, args, root, { fieldASTs }) => {
+      const alias = fieldASTs[0].alias && fieldASTs[0].alias.value
+      const key = alias || fieldASTs[0].name.value
```

Comment:
Instead of pulling the key out the schema descriptions, we're pulling it from the AST (allowing us to check for aliases at resolve-time). We use `key` here to find the right data from the `ctx.res` response object.